app/views/pages/_contact.haml

```diff
@@ -13,22 +13,22 @@
         =form_tag '/contact', method: :post, remote: true, id: 'contact_form' do
           .row.control-group
             .form-group.col-xs-12.floating-label-form-group.controls
-              =label_tag 'name', 'Name'
-              =text_field_tag 'name', nil, placeholder: "Name", class: 'form-control',
+              =label_tag 'name', t('name')
+              =text_field_tag 'name', nil, placeholder: t('name'), class: 'form-control',
                 required: "required", "data-validation-required-message" => t('contact_enter_name')
           .row.control-group
             .form-group.col-xs-12.floating-label-form-group.controls
-              =label_tag 'email', 'Email Address'
-              =text_field_tag 'email', nil, placeholder: "Email Address", class: 'form-control',
+              =label_tag 'email', t('email_address')
+              =text_field_tag 'email', nil, placeholder: t('email_address'), class: 'form-control',
```

Comment:
The = symbol should have one space separating it from code<br>Line is too long. [99/80]